src/ChainedMap.js

```diff
@@ -72,6 +72,13 @@ module.exports = class extends Chainable {
     return this.store.get(key);
   }
 
+  getOrCompute(key, fun) {
```

Comment:
Nit: can you rename this variable to `f` or `fn`?

Reply:
Done :smile: